packages/nestjs/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@sentry/nestjs",
+  "version": "8.11.0",
+  "description": "Official Sentry SDK for NestJS",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/nestjs",
+  "author": "Sentry",
+  "license": "MIT",
+  "engines": {
+    "node": ">=14.18"
```

Comment:
We should align this with the minimum supported version of nest since we also don't test for 14.

Reply:
done